### PR TITLE
Fixed notary version

### PR DESCRIPTION
**Pull request by dolgopolovwork**

### CODE REVIEW

File: src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt [hunk 1]
```diff
@@ -138,6 +138,20 @@ open class ChainAdapter(
...
     }
 
+    /**
+     * Returns raw query response with Iroha block.
+     * May be used to handle Iroha error codes manually
+     * @param queryAPI - Iroha queries network layer
+     * @param height - height of Iroha block to get
+     * @return Iroha block
+     */
+    private fun getBlockRawResponse(queryAPI: QueryAPI, height: Long): QryResponses.QueryResponse {
```
Comment (Mingela) [inline src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt]: why do you need it? queryAPI throws exceptions on errors

Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt]: Yep, you can use IrohaQueryHelperImpl.getBlock() and catch an exception instead these function.

Comment (Alexey-N-Chernyshov) [reply src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt]: I think, there is no need in this function.

Comment (dolgopolovwork) [reply src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt]: Fixed

**Pull request merged**